src/__tests__/CounterLabel.tsx

```diff
@@ -6,6 +6,8 @@ import {colors} from '../theme'
 import {render as HTMLRender, cleanup} from '@testing-library/react'
 import {axe, toHaveNoViolations} from 'jest-axe'
 import 'babel-polyfill'
+import {default as primitives} from '@primer/primitives'
```

Comment:
Instead of importing from primitives we can import the theme:

```diff
- import {default as primitives} from '@primer/primitives'
+ import theme from '../theme'
```

That might make values easier to reference in the tests